its/src/test/java/com/sonar/it/csharp/MetricsTest.java

```diff
@@ -50,6 +53,14 @@
   private static RuleChain getRuleChain() {
     assertThat(SystemUtils.IS_OS_WINDOWS).withFailMessage("OS should be Windows.").isTrue();
 
+    String localappData = System.getenv("LOCALAPPDATA") + "\\Temp\\.sonarqube";
```

Comment:
Scanner for MSBuild caches the analyzer, so running the test twice in a row means the old binary is used. This code deletes the cache, but there should be a way to run without cache. Ticket: https://jira.sonarsource.com/browse/SONARMSBRU-346

Reply:
Can you add this comment directly in the code? I am sure we will forget about it by the next time we work on this.